### DISCUSSION TITLE
can you add Russian to your launcher

### DISCUSSION THREAD

**Discussion by MrSmi1ey** [modrinth/.github#158] [Feature requests]
can you add Russian to your launcher

можете ли вы добавить русский язык в свой лаунчер

**Comment by brawaru** [comment]
Hey! Привет! <img src="https://cdn.discordapp.com/emojis/1116112242853814414.png?size=32" alt="Modrinth robot waving emoji" height="18" align="center" /> Modrinth is looking to add app localisation in the future. However, at the moment, our focus is on implementing other features and enhancing the overall app experience. That being said, we've recently started testing localisation on the site, and you're more than welcome to join the Russian translators team on [our Discord server](https://discord.gg/modrinth) and contribute translations on [Crowdin](https://crowdin.com/project/modrinth).

**Comment by MrSmi1ey** [comment]
Thank you for answering my question
